pillarbox-demo/build.gradle.kts

```diff
@@ -47,6 +44,18 @@ android {
                 "proguard-rules.pro"
             )
         }
+        flavorDimensions += "version"
```

Comment:
⚠️ **[detekt]** <detekt.StringLiteralDuplication> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Multiple occurrences of the same string literal within a single file detected. Prefer extracting the string literal into a property or constant.